assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -127,7 +149,7 @@ function PageAttachmentPanel() {
 
       {displayWarning && (
         <Row>
-          <Error>
+          <Error forwardedAs="span" size={PRESET_SIZES.X_SMALL}>
```

Comment:
I wonder if this should use the input's Errored version instead with the warning message displayed as the `Hint`. This warning is not in the design currently (I think, couldn't find it at least), so not sure if there are changes needed. Might be good to confirm with Sam (can be followed up in a separate PR though if any changes are needed)

Reply:
I retained the existing functionality for regression.

Reply:
Makes sense, thought of the style mainly but it's probably good as is! 👍

Reply:
That makes sense. Easy add too, I'll include it. :)